storage/notifications.go

```diff
@@ -217,12 +195,6 @@ func (b *BucketHandle) DeleteNotification(ctx context.Context, id string) (err e
 	ctx = trace.StartSpan(ctx, "cloud.google.com/go/storage.Bucket.DeleteNotification")
 	defer func() { trace.EndSpan(ctx, err) }()
 
-	call := b.c.raw.Notifications.Delete(b.name, id)
-	setClientHeader(call.Header())
-	if b.userProject != "" {
-		call.UserProject(b.userProject)
-	}
-	return run(ctx, func() error {
-		return call.Context(ctx).Do()
-	}, b.retry, true, setRetryHeaderHTTP(call))
+	opts := makeStorageOpts(true, b.retry, b.userProject)
```

Comment:
Hmmmm so the retry conformance tests are flagging Delete; however I can't see any difference in retry behavior either here or in the http_client.go implementation. Any idea what's going on?

Reply:
I just saw that too and was going to ask the same question haha

Reply:
Fixed.